### PR TITLE
{bio}[foss/2022a] EpiSCORE v0.9.5-20220621 w/ R 4.2.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/0add31f0a64cf174a36fb5d1dcb3d382 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/e/EpiSCORE/EpiSCORE-0.9.5-20220621-foss-2022a-R-4.2.1.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'RPackage'
+
+name = 'EpiSCORE'
+local_commit = '4f7daef'
```
Comment (akesandgren) [inline easybuild/easyconfigs/e/EpiSCORE/EpiSCORE-0.9.5-20220621-foss-2022a-R-4.2.1.eb:4]: I don't think this isn't guaranteed to be unique, better to use full commit string.

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/e/EpiSCORE/EpiSCORE-0.9.5-20220621-foss-2022a-R-4.2.1.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/e/EpiSCORE/EpiSCORE-0.9.5-20220621-foss-2022a-R-4.2.1.eb:4]: We use the short commit in every single easyconfig that uses local_commit, so i don't think we should start enforcing it here on a repo with 12 commits (with 7 characters still gives 78e9 combinations which assuming a repo has a thousand commits would mean a 1 in which even at 7 character is still 78e9 combinations

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**